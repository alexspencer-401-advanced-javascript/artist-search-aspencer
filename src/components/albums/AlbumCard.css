.AlbumCard {
  text-align: center;
  font-size: 20px;
  color: rgb(66, 66, 66);
  font-family: Arial, Helvetica, sans-serif;
  border-bottom: 1px solid darkgrey;

  img {
    height: 300px;
    width: 300px;
  }

  a {
    text-decoration: none;
  }
}