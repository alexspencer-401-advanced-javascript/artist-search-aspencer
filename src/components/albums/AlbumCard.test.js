import React from 'react';
import { shallow } from 'enzyme';
import AlbumCard from './AlbumCard';

describe('AlbumCard component', () => {
  it('renders AlbumCard', () => {
    const wrapper = shallow(<AlbumCard id={'test string'} title={'test string'} image={'test string'} name={'test string'}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
