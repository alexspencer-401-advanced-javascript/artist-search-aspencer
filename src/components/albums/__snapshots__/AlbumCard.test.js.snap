// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlbumCard component renders AlbumCard 1`] = `
<div
  className="AlbumCard"
  key="test string"
>
  <Link
    to="/songs/test string/test string"
  >
    <p
      className="AlbumCard"
    >
      test string
    </p>
  </Link>
  <img
    src="test string"
  />
</div>
`;
